### DISCUSSION TITLE
routes

### DISCUSSION THREAD

**Discussion by Xlebabutti** [requarks/wiki#6408] [Help / Questions] [answered]
Task:
Add a '.' to the url of the page, for example:
/home/en/page.1/test

If you change the regular expression 'filenamePattern' and remove it from checks, you can create such a page, but routing breaks down
There is no redirect and the page is available only at the link :/home/en/page.1/test.html (md)

Question
Perhaps there are other ways to solve this problem and add your own routes on the server side?
I ask for advice, thank you

**Comment by borepstein** [comment]
What kind of routing are you referring to? Routing within the virtual directory structure?

**Comment by NGPixel** [comment, marked as answer]
Don't use dots in your paths. The only dot that should be present is when using assets for the filename extension.
